### ISSUE TITLE
Compile Error -- reference type mismatched

### ISSUE THREAD

**Issue by Bo-Yuan-Huang** [Bo-Yuan-Huang/ILA-Tools#31]
**Describe the bug**
Fail under Apple LLVM when compiling stream buffer in unit tests.

**To Reproduce**
1. Build under OSX and using default build toolchain. 

**Expected behavior**
``` bash
/Users/travis/build/Bo-Yuan-Huang/ILA-Tools/test/unit-src/../unit-include/stream_buffer.h:13:42: error: reference to type 'const std::string' (aka 'const basic_string<char, char_traits<char>, allocator<char> >') could not bind to an lvalue of type 'const char [7]'
Ila GetStrmBuffSpec(const std::string& name = "SBSpec");
```

**Desktop (please complete the following information):**
 - OS: OS X 10.13.3
 - Compiler: Apple LLVM version 9.1.0 (clang-902.0.39.2)

**Comment by Bo-Yuan-Huang** [comment]
Will be fixed in branch Port.

**Comment by Bo-Yuan-Huang** [comment]
Fixed in cbbb216
